Resources/public/scss/modules/content-tree/_list.item.scss

```diff
@@ -0,0 +1,57 @@
+$list-item: '.c-list-item';
+
+#{$list-item} {
+    &__label {
+        display: flex;
+        align-items: center;
+    }
+
+    &__toggler {
+        display: inline-block;
+
+        &:after {
+            width: 0;
+            height: 0;
+            border-style: solid;
+            border-width: 8px 6px 0 6px;
+            border-color: $ez-black transparent transparent transparent;
+            transform: rotate(180deg);
+            transition: transform 0.2s $ez-admin-transition;
+            display: inline-block;
+            margin-left: calculateRem(8px);
+        }
+    }
+
+    .c-list {
+        opacity: 0;
+        list-style: none;
+        max-height: 0;
+        overflow: hidden;
+        transition: max-height 0.3s $ez-admin-transition, opacity 0.2s $ez-admin-transition;
+    }
+
+    &--has-sub-items {
+        > #{$list-item}__label {
+            #{$list-item}__toggler {
+                &:after {
+                    content: '';
+                }
+            }
+        }
+    }
+
+    &--is-expanded {
+        > .c-list {
+            opacity: 1;
+            max-height: calculateRem(20000px);
+        }
+
+        > #{$list-item}__label {
+            #{$list-item}__toggler {
```

Comment:
This is not compliant with BEM. We should try to avoid exceptions and use BEM whenever possible.

Reply:
Yes, it follows BEM convention. You need to understand what is going on here. Basically, I want to change the state of a toggler of the direct children of an item that has been toggled. I don't want to change the states of all togglers of element's children. I hope it's clear now :)

Reply:
I understand idea behind it, but still, I think that BEM assumes using only _class_ selectors. I thought that we are making exceptions in _admin-ui_ because of Bootstrap.

Reply:
In this case, it's also a correct BEM approach.